app/hackathon_variables.py

```diff
@@ -40,8 +40,10 @@
                    'the opening ceremony will be at 7:00 pm.'
 
 # (OPTIONAL) When to arrive at the hackathon
-HACKATHON_LEAVE = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. ' \
-                  'However the projects demo fair will be held in the morning from 10:30 AM to 1 PM.'
+HACKATHON_LEAVE_HACKER = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. ' \
+                         'However the projects demo fair will be held in the morning from 10:30 AM to 1 PM.'
+HACKATHON_LEAVE_OTHER = 'Closing ceremony will be held on Sunday October 15th from 3:00 PM to 5:00 PM. '
```

Comment:
Sponsors may also be interested in demos. Let's keep it.

Reply:
ok